pkg/transformers/image_test.go

```diff
@@ -166,14 +198,33 @@ func TestImageTagTransformer(t *testing.T) {
 						},
 					},
 				},
+				"spec3": map[string]interface{}{
+					"template": map[string]interface{}{
+						"spec": map[string]interface{}{
+							"initContainers": []interface{}{
+								map[string]interface{}{
+									"name":  "postgresdb",
+									"image": "my-postgres:v3",
+								},
+								map[string]interface{}{
+									"name":  "init-docker",
+									"image": "my-docker@sha256:25a0d4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3",
+								},
+							},
+						},
+					},
+				},
 			}),
 	}
 
-	it, err := NewImageTagTransformer([]types.ImageTag{
+	it, err := NewImageTransformer([]image.Image{
 		{Name: "nginx", NewTag: "v2"},
 		{Name: "my-nginx", NewTag: "previous"},
 		{Name: "myprivaterepohostname:1234/my/image", NewTag: "v1.0.1"},
 		{Name: "foobar", Digest: "sha256:24a0c4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3"},
+		{Name: "alpine", NewName: "myprivaterepohostname:1234/my/cool-alpine"},
```

Comment:
Thank you for adding those tests. 
Can you also add tests to cover different cases of image names, for example
`gcr.io/my-project/my-app-image`?

Reply:
you're welcome!
I added a couple of more test cases